### DISCUSSION TITLE
Feature: sitemap.xml出力とping送信機能

### DISCUSSION THREAD

**Discussion by toonaoki** [growilabs/growi#7711] [Ideas / Requests]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |Synology DSM  6.2.2-24922 Update 3|
|GROWI     |3.5.13|
|node.js   |10.16.3|
|npm       |6.9.0|
|Using Docker|no|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

### Client

| item     | version |
| ---      | --- |
|OS        |Windows10|
|browser   |Chrome 77.0.3865.90|

What is the expected result? (期待される動作)
-------------------------------------------
- sitemap.xmlの自動生成
- 更新したタイミングで任意のURLへのpingの送信

Note
----
はじめまして。toonaokiと申します。
便利に使わせていただいています。
NAS上のDockerでうまく動いているので既存の公開用のWikiも置き換えていきたいと考えています。
用途的にはローカル利用が主なのかもしれませんが、サイトマップが投げられると検索性も高められるのかなと思いました。
他のWikiだとプラグインなどで対応していますが、この快適さに慣れるとちょっと戻れないなというのが正直なところです。
ご検討よろしくお願いします。

**Comment by yuki-takei** [comment]
@toonaoki コメントが遅くてすみません。ご意見ありがとうございます。

GROWI はオープンWikiとしても活用してもらいたいなと考えて開発しているので、仰るようにサイトマップや OGP の拡充は必要だと思います。

現状のロードマップだと開発のための開発が重めに走っているので、優先順はだいぶ後になるかもしれませんが、需要次第で早めに実装できるよう努力したいと思います。

因みに優先順は Twitter で呟いていたりします → https://twitter.com/weseek_inc/status/1186233553526198272
